### ISSUE TITLE
[FEATURE]: Create the Tooltip component for the Storybook

### ISSUE THREAD

**Issue by psanders** [fonoster/fonoster#677]
### Is your feature request related to a problem?

This component is part of a reusable design system for Fonoster's dashboard. It helps achieve consistency and reduces repetitive implementation work.

<img width="197" alt="Tooltip Example" src="https://github.com/user-attachments/assets/2d57dfa3-68e8-4c2a-b6a5-df95efc3cbd5" />

### Describe the solution you'd like

The Tooltip should follow the design in the Figma file and be implemented as a standalone React component with the following props: 

The closest component to this in MUI is the [Tooltip](https://mui.com/material-ui/react-tooltip/).

(Note: MUI’s Tooltip doesn’t seem to support multiline text, which is a requirement for our implementation)

### Describe alternatives you've considered

Building the component from scratch if using MUIs as a base is impractical.

### Additional context

Before working on this item, please check the [CONTRIBUTING](https://github.com/fonoster/fonoster/blob/main/CONTRIBUTING.md) document. Details about the Tooltip component can be found [here.](https://www.figma.com/design/OsZlne0RvIgoFlFKF7hnAU/Shared-Component-Library?node-id=947-2396&m=dev)

Acceptance Criteria: 

  - [ ] Matches Figma design
  - [ ] Storybook story with documentation
  - [ ] Includes basic unit tests (if applicable)

**Comment by obrucheoghene** [comment]
Hi @psanders 
I will like to take up this next

**Comment by psanders** [comment]
Ofc! All yours, friend.

**Comment by obrucheoghene** [comment]
Hi  @psanders 
> The Tooltip should follow the design in the Figma file and be implemented as a standalone React component with the following props:

The props weren't stated in the issue, here is what I am proposing

```
type TooltipProps = {
    content: string | string[],  // array of string for multiline text 
    placement?: TooltipPlacement,
    children: React.ReactNode
}

type TooltipPlacement =
    | 'top'
    | 'right'
    | 'bottom'
    | 'left'
  ```
  
Is this okay by you ?

**Comment by psanders** [comment]
@obrucheoghene 

Good catch. What would the children mean in this instance?

**Comment by obrucheoghene** [comment]
The children is the component that the tip will be on

example
```
  <Tooltip content="Request Video Call" placement="top">
       <Button>Video Call</Button> // button is children
   </Tooltip>
```

**Comment by psanders** [comment]
@obrucheoghene Thanks for clarifying. Makes perfect sense. 

I agree with your proposed props.

**Comment by psanders** [comment]
Closed by #685
